src/Controller/AdminController.php

```diff
@@ -32,15 +32,20 @@ public function showStatistics(
         $exhibition = null;
         if ($id !== null) {
             $exhibition = $exhibitionRepository->find($id);
+        } elseif ($id === null) {
+            $exhibition = $exhibitionRepository->findOneBy([]);
```

Comment:
No you shouldn't manage this here but you better should ask the exhibition directly in the params of the action